modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/CalciteQueryProcessor.java

```diff
@@ -186,8 +197,10 @@ public class CalciteQueryProcessor extends GridProcessorAdapter implements Query
     private final FrameworkConfig frameworkCfg;
 
     /** Query planner timeout. */
-    private final long queryPlannerTimeout = getLong(IGNITE_CALCITE_PLANNER_TIMEOUT,
-        DFLT_IGNITE_CALCITE_PLANNER_TIMEOUT);
+    private final long qryPlannerTimeout = getLong(IGNITE_CALCITE_PLANNER_TIMEOUT, DFLT_IGNITE_CALCITE_PLANNER_TIMEOUT);
+
+    /** Use query blocking task executor. */
+    private final boolean useQryBlockingTaskExecutor = getBoolean(IGNITE_CALCITE_USE_QUERY_BLOCKING_TASK_EXECUTOR);
```

Comment:
`useQryBlockingTaskExecutor` has single usage. Can be inlined.

Reply:
Fixed